src/rust/engine/src/core.rs

```diff
@@ -99,6 +99,23 @@ impl Key {
   }
 }
 
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default)]
+pub struct RunnableComplete {
+  value: Value,
+  is_throw: bool,
+}
+
+impl RunnableComplete {
+  pub fn value(&self) -> &Value {
```

Comment:
You can skip making accessors if you mark the struct fields 'pub'... for simple structs like this, accessors generally seem premature, and I've been avoiding them.

```
pub struct RunnableComplete {
  pub value: Value,
  ..
}
```